shared/unattended/Fedora-22.ks

```diff
@@ -0,0 +1,36 @@
+install
+KVM_TEST_MEDIUM
+text
+reboot
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+autopart
+
+%packages
+@standard
```

Comment:
It's not available in the "Server" spin. I thought of changing the ISO/URL to "Workstation", but seemed adequate. At least to the subset of tests I ran with those guests.